### ISSUE TITLE
loader script: sourcemaps must be public

### ISSUE THREAD

**Issue by mustafa0x** [getsentry/sentry-docs#7443]
### Core or SDK?

Core Sentry product

### Which part? Which one?

browser

### Description

https://docs.sentry.io/platforms/javascript/install/loader/#available-bundles says:

> To have correct stack traces for minified asset files when using the Loader Script, you will have to upload & host your Source Maps publicly.

Can't this be solved by simply by setting a release after using the loader script? eg

```js
Sentry.onLoad(function() {
    Sentry.init({
        release: " ... ",
    });
});
```

The loader script is a tiny wrapper around the main package, so imposing this limitation doesn't seem to make much sense?

### Suggested Solution

-

**Comment by mustafa0x** [comment]
OT, but easier than opening a new issue:

https://docs.sentry.io/platforms/javascript/usage/#capturing-messages

> Typically messages are not emitted, but they can be useful for some teams.

"but they can be" doesn't make sense. Probably what is meant is "that can be".

**Comment by Lms24** [comment]
Hi @mustafa0x, thanks for writing in and apologies for the late reply!

> Can't this be solved by simply by setting a release after using the loader script

Either you [host your source maps](https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/hosting-publicly/) so that we can access them from your server or you have to [upload them](https://docs.sentry.io/platforms/javascript/sourcemaps/#uploading-source-maps) to Sentry. In recent SDK versions, setting a release is not required anymore to get source maps working.  

I agree though that 

> you will have to upload & host your Source Maps publicly

is not entirely correct, as it implies that with the loader, you can only host them which is not true. Since you already looked into this, would you mind opening a PR that basically replaces the "and" with an "or"? Of course, we want to link to the respective docs, too.

**Comment by Lms24** [comment]
> Typically messages are not emitted, but they can be useful for some teams

I find this sentence quite confusing, tbh. I think we want to reword this to something like

> "Typically, our SDKs don't automatically capture messages. However, it might be useful for teams to capture certain messages."

**Comment by Lms24** [comment]
@mustafa0x I just went ahead and changed the sentences as I was looking at the docs anyway. Thanks for providing feedback!

**Comment by mustafa0x** [comment]
Thanks a ton Lukas! I was a bit wary of making that change, in case I didn't get it right. 😅
